client/src/pages/game_lobby/team_select/TeamSelectActions.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import AddCircle from "@material-ui/icons/AddCircle";
+import IconButton from "@material-ui/core/IconButton";
+import RemoveCircle from "@material-ui/icons/RemoveCircle";
+
+export const JoinRoleAction = (props) => {
+  return (
+    <IconButton
+      onClick={() => {
+        props.joinRole(props.teamCode, props.index);
+      }}
+      edge="end"
+    >
+      <AddCircle />
+    </IconButton>
+  );
+};
+
+export const LeaveRoleAction = (props) => {
```

Comment:
This is just a suggestion, but I like to use object destructuring when reaching into props. The downside of this is that it could run into some variable naming overlap 

```suggestion
export const LeaveRoleAction = ({teamCode, index) => {
```